yb-voyager/cmd/analyzeSchema.go

```diff
@@ -374,12 +376,54 @@ func checkStmtsUsingParser(sqlInfoArr []sqlInfo, fpath string, objType string) {
 			continue
 		}
 
-		if objType == TABLE {
-			createTableNode, ok := parseTree.Stmts[0].Stmt.Node.(*pg_query.Node_CreateStmt)
-			if ok {
-				reportGeneratedStoredColumnTables(createTableNode, sqlStmtInfo, fpath)
-			}
+		createTableNode, isCreateTable := parseTree.Stmts[0].Stmt.Node.(*pg_query.Node_CreateStmt)
+		alterTableNode, isAlterTable := parseTree.Stmts[0].Stmt.Node.(*pg_query.Node_AlterTableStmt)
+		createIndexNode, isCreateIndex := parseTree.Stmts[0].Stmt.Node.(*pg_query.Node_IndexStmt)
+
+		if objType == TABLE && isCreateTable {
+			reportGeneratedStoredColumnTables(createTableNode, sqlStmtInfo, fpath)
+		}
+		if isAlterTable {
+			reportAlterTableVariants(alterTableNode, sqlStmtInfo, fpath, objType)
+		}
+		if isCreateIndex {
+			reportCreateIndexStorageParameter(createIndexNode, sqlStmtInfo, fpath)
+		}
+
+	}
+}
+
+func reportCreateIndexStorageParameter(createIndexNode *pg_query.Node_IndexStmt, sqlStmtInfo sqlInfo, fpath string) {
+	indexName := createIndexNode.IndexStmt.GetIdxname()
+	summaryMap["INDEX"].invalidCount[sqlStmtInfo.objName] = true
+	if len(createIndexNode.IndexStmt.GetOptions()) > 0 {
```

Comment:
There seem to be multiple storage parameters one can set on a CREATE INDEX. https://www.postgresql.org/docs/current/sql-createindex.html

Have we verified that none of those are supported by YB?

Reply:
Yes so AFAIK YB doesn't have support for any storage parameters If you see CREATE TABLE storage parameters docs [here](https://docs.yugabyte.com/preview/api/ysql/the-sql-language/statements/ddl_create_table/#storage-parameters-1), it is mentioned that all of them are ignored and there is no documentation even, for CREATE index on docs, so I believe none of them are supported for it as well by YB but we are erroring out in this case which is not consistent with CREATE TABLE.

Reply:
Got it! Pls add link to this documentatin in code as a comment

Reply:
Just to confirm: You pointed to `CREATE TABLE` docs. This is `CREATE INDEX` though, and here YB fails, is that correct?